src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx

```diff
@@ -18,188 +18,181 @@ Our Sendmail integration monitors the performance of your Sendmail mail transfer
+                split_by: \s+(\d+)\s+(.*.)
+
+            #Read the senders with message count.
+            - event_type: SendmailSendersbyMessageCount
+            commands:
+                - run: cat /var/log/mail.log | grep "from=<.*@.*>" | awk '{print $7}' | cut -d "<" -f2 |cut -d ">" -f1 | sort | uniq -c
+                split: horizontal
+                header_split_by: \s+(\d+)\s+(.*.)
+                set_header: [msg.total, senders.total]
+                regex_match: true
+                split_by: \s+(\d+)\s+(.*.)
+                - run: cat /var/log/mail.log | grep "from=<.*@.*>" | grep "$(date +'%b %e')" | awk '{print $7}' | cut -d "<" -f2 |cut -d ">" -f1 | sort | uniq -c
+                split: horizontal
+                header_split_by: \s+(\d+)\s+(.*.)
+                set_header: [msg.latest, senders.latest]
+                regex_match: true
+                split_by: \s+(\d+)\s+(.*.)
+
+            #Read the senders host/domain.
+            - event_type: SendmailSendersHostname
+            commands:
+                - run: cat /var/log/mail.log | grep "from=<.*.>" | awk '{print $7}' | grep -oE '[^@]+[a-zA-Z0-9\-\.]+\.[a-zA-Z]{2,}' | sort | uniq -c
+                split: horizontal
+                header_split_by: \s+(\d+)\s+(.*.)
+                set_header: [hostCount.total, hostname.total]
+                regex_match: true
+                split_by: \s+(\d+)\s+(.*.)
+                - run: cat /var/log/mail.log | grep "from=<.*.>" | grep "$(date +'%b %e')" | awk '{print $7}' | grep -oE '[^@]+[a-zA-Z0-9\-\.]+\.[a-zA-Z]{2,}' | sort | uniq -c
+                split: horizontal
+                header_split_by: \s+(\d+)\s+(.*.)
+                set_header: [hostCount.latest, hostname.latest]
+                regex_match: true
+                split_by: \s+(\d+)\s+(.*.) 
+    ```   
+
+    </Step>
+    <Step>
 ## Forward your Sendmail logs to New Relic [#forward-logs]
 
-You can use our [log forwarding](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/) to forward Sendmail logs to New Relic.
-
-On Linux machines, your log file named `logging.yml` should be present in this path:
-
-```shell
-/etc/newrelic-infra/logging.d/
-```
-
-Add the following script to the `logging.yml` file:
-
-```yml
-  
-logs:
-  - name: mail.log
-    file: /var/log/mail.log
-    attributes:
-      logtype: sendmail_log
-  - name: mail.err
-    file: /var/log/mail.err
-    attributes:
-      logtype: sendmail_error
-```
-
-
+You can use our [log forwarding](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/) capability to forward Sendmail logs to New Relic.
+
+Add the following script to `/etc/newrelic-infra/logging.d/logging.yml`:
+
+    ```yml
+    
+    logs:
+    - name: mail.log
+        file: /var/log/mail.log
+        attributes:
+        logtype: sendmail_log
+    - name: mail.err
+        file: /var/log/mail.err
+        attributes:
+        logtype: sendmail_error
+    ```        
```

Comment:
in the preview, this is appearing in the codeblock. very small buggy behavior, could be related to indents?

Reply:
that is, "this" == the closing ```